doc/release-process.md

```diff
@@ -41,7 +41,12 @@ Check out the source code in the following directory hierarchy.
 	pushd ./bitcoin
 	export SIGNER=(your Gitian key, ie bluematt, sipa, etc)
 	export VERSION=(new version, e.g. 0.8.0)
+	git fetch
 	git checkout v${VERSION}
+
+  Add other gitian builders keys to your gpg keyring
+
+	gpg --import ./contrib/gitian-downloader/*.pgp
```

Comment:
I think the verification/import keys part should go to a section below... maybe a new one.